### ISSUE TITLE
Merge Theme Systems

### ISSUE THREAD

**Issue by KaiTheRedNinja** [AuroraEditor/AuroraEditor#276]
### Is your feature request related to a problem? Please describe.

Merge Theme, HighlightTheme and AuroraTheme

### Describe the solution you'd like

![image](https://user-images.githubusercontent.com/88234730/194495428-5745fed0-957e-4973-8941-e69face69a3f.jpeg)

### Describe alternatives you've considered

nil

### Additional context

nil

**Comment by KaiTheRedNinja** [comment]
This was completed in #227 and is no longer needed after #283 as the old theme system has now been fully replaced.
